apps/web/src/pages/templates/components/VariantsListSidebarActions.tsx

```diff
@@ -0,0 +1,171 @@
+import { Group } from '@mantine/core';
+import { useEffect, useRef, useState } from 'react';
+import { useFieldArray, useFormContext } from 'react-hook-form';
+import { useLocation, useNavigate, useParams } from 'react-router-dom';
+import { FilterPartTypeEnum } from '@novu/shared';
+import { ActionButton, Condition, ConditionPlus, Trash, VariantPlus } from '@novu/design-system';
+
+import { Conditions, IConditions } from '../../../components/conditions';
+import { When } from '../../../components/utils/When';
+import { useEnvController } from '../../../hooks';
+import { useBasePath } from '../hooks/useBasePath';
+import { useFilterPartsList } from '../hooks/useFilterPartsList';
+import { useStepIndex } from '../hooks/useStepIndex';
+import { useStepVariantsCount } from '../hooks/useStepVariantsCount';
+import { DeleteConfirmModal } from './DeleteConfirmModal';
+import { IForm } from './formTypes';
+import { makeVariantFromStep, useTemplateEditorForm } from './TemplateEditorFormProvider';
+
+const variantsCreatePath = '/variants/create';
+
+export const VariantsListSidebarActions = () => {
```

Comment:
these are the actions shown in the variants preview header, extracted from the existing code